ot-node.js

```diff
@@ -341,6 +341,34 @@ class OTNode {
         await autoUpdaterCommand.execute();
     }
 
+    initializeParanets() {
+        const paranetIdService = this.container.resolve('paranetIdService');
+        const blockchainModuleManager = this.container.resolve('blockchainModuleManager');
+        const tripleStoreService = this.container.resolve('tripleStoreService');
+        const tripleStoreModuleManager = this.container.resolve('tripleStoreModuleManager');
+        const invalidParanets = [];
+        const validParanets = [];
+        this.config.assetSync?.syncParanets.forEach((paranetId) => {
+            if (!paranetIdService.isUAL(paranetId)) {
+                invalidParanets.push(paranetId);
+                this.logger.warn('');
```

Comment:
Put here some message, like unable to initialize Paranet with id because of invalid UAL format